Makefile

```diff
@@ -167,7 +167,7 @@ clean:
 	$(RM) -r ebin
 
 .PHONY: distclean
-distclean: clean
```

Comment:
I disagree. I took a look at the following guidelines and I think the current split between `distclean` and `maintainer-clean` is more reasonable.

https://www.gnu.org/prep/standards/html_node/Standard-Targets.html

What do you think?

Reply:
OK.  I can revert this.